net.sf.eclipsecs.core/src/net/sf/eclipsecs/core/config/meta/MetadataFactory.java

```diff
@@ -504,6 +719,20 @@ private static void processModules(Element groupElement, RuleGroupMetadata group
         String messageKey = quickfixEl.attributeValue(XMLTags.KEY_TAG);
         module.addMessageKey(messageKey);
       }
+
+      // Update the metadata with module details from checkstyle
+      if (internalName.equals("EmptyPublicCtorInClassCheck")
+              || !(module.getGroup().getGroupName().equals("SevNTU checks")
+              || module.getGroup().getGroupName().equals("My custom checks")
```

Comment:
no hardcodes please for any thirdparty modules.

Reply:
dONE